ocp3/ocp3/GameManager.swift

```diff
@@ -94,8 +94,122 @@ class GameManager {
         team2 = makeTeam()
         team2?.teamInfo()
     }
+    ///////////////////////////////////Fonctions pour gerer les combats /////////////////////////////////////////
+    func attack(charAttacker: Character, charTarget: Character) {
+        
+        charTarget.damage(attacker: charAttacker)
+        print("\(charAttacker.name) a attaqué \(charTarget.name).Il lui reste \(charTarget.life) hp.")
+    }
+    
+    ///////////////////////////////////Fonction qui permet de faire une boucle jusqu'à ce qu'il y ait un vainqueur
+    
+    func play() {
+        var attackerTeam = team1!
+        var targetTeam = team2!
+        repeat {
+            let attacker = attackerTeam.chooseAttacker()
+            let target = targetTeam.chooseTarget()
+            attack(charAttacker: attacker, charTarget: target)
+            swap(&attackerTeam, &targetTeam)
+        } while team1!.alive(team: team1!) == false && team2!.alive(team: team2!) == false
+        if team1!.alive(team: team1!) == true {
```

Comment:
Maybe we can extract this code to a new private fonction, WDYT (what do you think)? Maybe `annonceWinner()` or something else?